docs/includes/ksql-includes.rst

```diff
@@ -67,7 +67,7 @@ Create and produce data to the Kafka topics ``pageviews`` and ``users``. These s
 -------------------
 Launch the KSQL CLI
 -------------------
-To launch the CLI, run the following command. It will route the CLI logs to the ``./ksql_logs`` directory. By default,
+To launch the CLI, run the following command. It will route the CLI logs to the ``.ksql_logs`` directory. By default,
```

Comment:
@joel-hamill : Typo in here.  It is not the `.ksql_logs` AFAICT.  Note the leading `.` (dot).  This would be a hidden directory.

Reply:
This should read:

> It will route the CLI logs to the `./ksql_logs` directory within the directory from which you are running the command. By default, ...